package.json

```diff
@@ -2,17 +2,17 @@
   "name": "react-colorful",
   "version": "2.3.1",
   "description": "A tiny color picker component for modern React apps",
-  "source": "src/hex.tsx",
+  "source": "src/index.ts",
   "main": "dist/index.js",
   "module": "dist/index.module.js",
   "esmodule": "dist/index.esmodule.js",
   "umd:main": "dist/index.umd.js",
-  "types": "dist/hex.d.ts",
+  "types": "dist/index.d.ts",
   "scripts": {
     "lint": "eslint src/**/*.{ts,tsx} demo/src/**/*.js",
     "size": "npm run build && size-limit",
     "test": "jest tests",
-    "build": "node ./build-packages.js",
+    "build": "microbundle build --entry src/index.ts --jsx React.createElement --css-modules true --sourcemap false --tsconfig tsconfig.build.json",
```

Comment:
as far as I know, we should add sideEffects key to package.json to mark the library as tree-shakable.
https://webpack.js.org/guides/tree-shaking/

Reply:
Definitely. We will have a side effect in the CSS but everything else should be side effect free.

Reply:
Done